src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -83,6 +83,8 @@ private void respondentDefaults(CaseData caseData) {
         if (caseData.getRespondentCollection() != null && !caseData.getRespondentCollection().isEmpty()) {
             RespondentSumType respondentSumType = caseData.getRespondentCollection().get(0).getValue();
             caseData.setRespondent(nullCheck(respondentSumType.getRespondentName()));
+            if (caseData.getEt3Received() == null)
```

Comment:
I would always include braces even if there is only one statement. See the Google Java Style Guide
https://google.github.io/styleguide/javaguide.html#s4.1-braces

Is it possible to do a unit test for this change?

Reply:
Yes please, add the braces as James suggested and some unit test to verify the change.
Also, this PR should be to R5.0.9 and nnot master. This is the current branch we have on demo.

Reply:
Done. I have added test with existing caseData "scotlandCcdRequest1".  et3Received has been not set in json so it is null. 
To assert that it doesn't set to "No" when it is not null, we have to create a new caseData json or update existing one.